test/cowmachine_test.erl

```diff
@@ -0,0 +1,52 @@
+-include_lib("eunit/include/eunit.hrl").
+
+-module(cowmachine_test).
+
+-export([
+    execute/2,
+    process/4
+]).
+
+cowmachine_start_test() ->
+    ok = application:ensure_started(ranch),
```

Comment:
You might also use `application:ensure_all_started(cowmachine)`?

Reply:
Then you get an error that ranch is not started... Maybe cowmachine.app.src has to list all applications for this.

Reply:
Ah, yes, it needs to list the other apps: cowboy, lager, zotonic_stdlib.

Was somehow never added.